playbooks/oxa_configuration.yml

```diff
@@ -2,10 +2,12 @@
 - name: Configure OXA
   hosts: all
   vars:
-    secrets_path: "{{ oxa-tools_config_path }}"
```

Comment:
fyi, i removed this unused variable in favor of a direct path to the environment-specific secrets called env_file